KsApi/models/templates/ProjectTemplates.swift

```diff
@@ -3,6 +3,15 @@ import Prelude
 // swiftlint:disable line_length
 extension Project {
   internal static let template = Project(
+    availableCardTypes: [
+      "AMEX",
+      "MASTERCARD",
+      "VISA",
+      "DISCOVER",
+      "JCB",
+      "DINERS",
```

Comment:
Note that `DINERS` is actually not a `supportedNetwork` according to Apple Pay, so it's filtered out.